src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -76,35 +77,24 @@ public void run() {
                     String requestJson = requestFilterQueue.poll(100, TimeUnit.MILLISECONDS);
                     if (requestJson != null) {
                         String requestId = getRequestIdFromRequestJson(requestJson);
-                        if (igoCmoRequestFilter && !isCmoRequest(requestJson)) {
-                            // skip non-cmo requests if the cmo filter is enabled
-                            LOG.info("CMO request filter enabled - skipping non-CMO request: "
-                                    + requestId);
-                            continue;
-                        }
-                        // does the request filtering...
+                        requestJson = validRequestChecker.checkIfValidRequest(requestJson);
```

Comment:
I think we should store the filtered request json in a different variable so that we can log differences between the two

it might make sense to move the json comparator from the consistency checker to the commons repo but we can hold off on that until we refine this idea a little more